dojo/templates/base.html

```diff
@@ -165,7 +165,7 @@
                             <li>
                                 <a href="{% url 'swagger-ui_oa3' %}?docExpansion=none" target="_blank"><i class="fa fa-book fa-fw" title="OpenAPI v3 Docs"></i> API v2 OpenAPI3 Docs </a>
                             </li>
-                            <li><a href="https://defectdojo.github.io/django-DefectDojo" target="_blank">
+                            <li><a href="{{ DOCUMENTATION_URL }}" target="_blank">
```

Comment:
There is another link to the documentation at the bottom of the page. Did you leave it unchanged on purpose?

Reply:
No. Thanks. I'm going to change it as well.

Reply:
Might make sense, only wanted to be sure it hasn't been forgotten 😃